src/modules/gauging-stations/controller.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const gaugingStationsService = require('../../lib/services/gauging-station-service');
+const controller = require('../../lib/controller');
+
+const getGaugingStation = async request =>
+  controller.getEntities(
+    request.params.gaugingStationId,
+    gaugingStationsService.getGaugingStations
+  );
+
+const getGaugingStationByRef = async request =>
+  controller.getEntities(
+    request.params.stationRef,
+    gaugingStationsService.getGaugingStationByRef
```

Comment:
Hmm... Could you double check this please: The service method name here is calling for `getGaugingStationByRef` but the exported method is `getGaugingStation**s**ByRef` (Plural vs Singular)